config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_vertexaiindexes.vertexai.cnrm.cloud.google.com.yaml

```diff
@@ -140,19 +140,15 @@ spec:
                     - dimensions
                     type: object
                   contentsDeltaUri:
-                    description: |-
-                      Allows inserting, updating  or deleting the contents of the Matching Engine Index.
-                      The string must be a valid Cloud Storage directory path. If this
-                      field is set when calling IndexService.UpdateIndex, then no other
-                      Index field can be also updated as part of the same call.
-                      The expected structure and format of the files this URI points to is
-                      described at https://cloud.google.com/vertex-ai/docs/matching-engine/using-matching-engine#input-data-format.
+                    description: "Allows creating or replacing the contents of the
+                      Matching Engine Index.\n\t\t\t\t\t\t\tConfig Connector always
```

Comment:
I think we should hide the TF details, but describe it from the functionality perspective. Something like:

```
Allows creating or replacing the contents of the Matching Engine Index. When being updated,
the existing content of the Index will be replaced by the data from the latest contentsDeltaUri.
The string must be a valid Cloud Storage directory ...
```

Reply:
Thank you for the suggestion. Updated.